### PR TITLE
Add Makefile and fix errors

### PR DESCRIPTION
**Pull request by localanu**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+#	or libXXX.a naming scheme. You can specify full paths or paths relative
+#	to the Makefile. The paths included are not parsed recursively, so
+#	include all of the paths where libraries must be found. Directories where
+#	source files were specified are	automatically included.
+LIBPATHS = 
+
+#	Additional paths to look for system headers. These use the form
+#	"#include <header>". Directories that contain the files in SRCS are
+#	NOT auto-included here.
+SYSTEM_INCLUDE_PATHS =  /boot/develop/headers/be /boot/develop/headers/cpp /boot/develop/headers/posix /boot/develop/lib /boot/beos/system/lib
```
Comment (mmuman) [inline Makefile]: Those should be removed, they are both unneeded and wrong for some of them.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+#		you can simply specify XXX for the library. (e.g. the entry for
+#		"libtracker.so" would be "tracker")
+#
+#	-	for GCC-independent linking of standard C++ libraries, you can use
+#		$(STDCPPLIBS) instead of the raw "stdc++[.r4] [supc++]" library names.
+#
+#	- 	if your library does not follow the standard library naming scheme,
+#		you need to specify the path to the library and it's name.
+#		(e.g. for mylib.a, specify "mylib.a" or "path/mylib.a")
+LIBS =  /boot/system/lib/libbe.so /boot/system/lib/libroot.so /boot/system/lib/libscreensaver.so /boot/system/lib/libtracker.so /boot/system/lib/libsupc++.so
```
Comment (mmuman) [inline Makefile]: Please simplify the naming, you should just need "be tracker screensaver $(STDCPPLIBS)" here (libroot is automatic).

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+#	Also note that spaces in folder names do not work well with this Makefile.
+SRCS =  ColorContainer.cpp Prefs.cpp SettingsView.cpp Panel.cpp Wolle.cpp
+
+#	Specify the resource definition files to use. Full or relative paths can be
+#	used.
+RDEFS = 
+
+#	Specify the resource files to use. Full or relative paths can be used.
+#	Both RDEFS and RSRCS can be utilized in the same Makefile.
+RSRCS =  FileInfo.rsrc
```
Comment (mmuman) [inline Makefile]: Please convert the rsrc to rdef and fix the makefile to use it.

**Comment by mmuman** [comment]
Ok I suppose you got it compiled already, I'd just add some fixes to the makefile.

**Comment by localanu** [comment]
roger!

**Comment by scottmc** [comment]
Maybe add a readme.md and maybe a screenshot if you can get one?  Also test it out and see if you detect any issues which may need to be addressed, or if you have ideas for future enchancements, and open a new issue for each.

**Comment by localanu** [comment]
@scottmc Added README.md

**Pull request merged**